### DISCUSSION TITLE
[Help]: Manually changing slide doesn't reset autoplay counter

### DISCUSSION THREAD

**Discussion by mohamedaleya** [davidjerleke/embla-carousel#964] [Help] [answered]
### Summary

right now, when i manually change carousel slide (let's say at 14s), it will be automatically changed just 1 second after because of the Autoplay. is there a way to reset autoplay counter when i manually change slide?

    <HeroCarousel
      opts={{
        loop: true,
        duration: 30,
      }}
      plugins={[
        Autoplay({
          delay: 15000,
        }),
        
      ]}
    >

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
Hi @mohamedaleya,

Yes, it’s done like this:

```tsx
emblaApi.plugins().autoplay?.reset()
```

Also see the `Autoplay` plugin [methods reference](https://www.embla-carousel.com/plugins/autoplay/#methods) section.

**Reply by mohamedaleya** [reply]
Thank you!
